Utils/mhubConnection.js

```diff
@@ -0,0 +1,32 @@
+const domain = require('domain')
+const {MClient} = require('mhub')
+const MsLogger = require('@first-lego-league/ms-logger').Logger()
+const {getCorrelationId, correlateSession} = require('@first-lego-league/ms-correlation')
+
+const MHUB_NODES = {
+  PUBLIC: 'public'
+}
+
+const MHUB_CLIENT_ID = 'cl-schedule'
+
+const mhubClient = new MClient(process.env.MHUB)
+mhubClient.on('error', msg => {
+  domain.create().run(() => {
+    correlateSession()
+    MsLogger.error('Unable to connect to mhub, other modules won\'t be notified changes \n ' + msg)
+  })
+})
+
+function publish(node, topic, msg) {
```

Comment:
All functions are using the same node, maybe it should'nt be an argument

Reply:
All the functions that uses this method now,
I want to keep it mostly generic for future use